core/audits/mainthread-work-breakdown.js

```diff
@@ -82,6 +83,15 @@ class MainThreadWorkBreakdown extends Audit {
     const settings = context.settings || {};
     const trace = artifacts.traces[MainThreadWorkBreakdown.DEFAULT_PASS];
 
+    let tbtSavings = 0;
+    try {
+      const metricComputationData = Audit.makeMetricComputationDataInput(artifacts, context);
+      const tbtResult = await TotalBlockingTime.request(metricComputationData, context);
+      tbtSavings = tbtResult.timing;
+    } catch (err) {
+      if (!/Could not find any top level events/.test(err)) throw err;
```

Comment:
Added this case to pass the "no events are present" case for this audit. In practices we shouldn't ever hit this error, and if we do then there are probably bigger problems than this audit. I'm open to removing that test case if it cleans up the code here a bit.

Reply:
Would it work to make this computed artifact return null in this case, and having callers that would find that exceptional throw themselves?

Reply:
I think that's a lot of `null` handling for an error that generally shouldn't happen. Plus we would have to generalize it for all computed metrics not just TBT since the result type is shared.

TBH I think it would be more reasonable to remove this logic and surface the error in the audit if the main thread doesn't have any tasks. The only reason this exception is made is for one of our test cases that asserts this audit returns no data if there are no tasks.